frontend/controllers/application_controller.js

```diff
@@ -48,6 +48,9 @@ export default class ApplicationController extends Controller {
   }
 
   setUserData(key, data) {
+    if(event.target.innerHTML == 'Delete view') {
```

Comment:
Now we're going to do this every time we set user data, that's not what we want.
There must be a way to isolate this bit and put it in action-table where it belongs.

A view means nothing to satis.

Reply:
agreed, Modified action table to handle this, hence closing this PR
Thank you.